### PR TITLE
Update travis config for auto deployments.

### PR DESCRIPTION
**Pull request by jettisonjoe**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,23 +1,30 @@
...
-  - "2.7"
+- '2.7'
 addons:
   apt:
     packages:
-      - swig
-      - libusb-1.0-0-dev
-      - libprotobuf-dev
-      - protobuf-compiler
+    - swig
```
Comment (fahhem) [inline .travis.yml]: why did you dedent all of these lists? It's actually a little confusing now (keys of a dict and lists for a key are at the same level now)

File: .travis.yml [hunk 2]
```diff
@@ -1,23 +1,30 @@
...
 after_success:
-  - coveralls
+- coveralls
+deploy:
+  provider: pypi
+  user: jettisonjoe
+  on:
+    tags: true
+  password:
+    secure: zFZNHtiht6OhCrbWMGGb+yBMmTJLKtAE5RiDl60m58Jx4t/UhMtHoC0+jtZ1hq2aeprfvw60zAIhB3HbLvb+E4Dcf9U8hEGDxJhMSMwGkXvFtrdAqce+I1YcNTXwCUApoJfE8Px1IZMRY9e1+u7r+cIWVggc3X97bWGGo+68A/ZkFxyGv75n8HRfCyHxK2hG/CGJWWmiK+fpDD68I3GbTyL1V5cRAkXNr4O1cSBDzeo0qOKzLZKAYR8M4SEQCz0Qlyatksa/p76ijF0tt5Q+kdVRANwlIltn57l43xwgL6ViOLs0rP19hlDITj5g6FZsNlRPPNxD5mUMuDEZefdcfm7sfvYHlU7hi38bxxWHXIbhUg6147V1+1oVu+dcxpv6/LgbYKSF2rxmkB6pZLlvOKK/U8Ni+3cN5KO1IumBdtQPDvSVUnsEB6GiVazm5xdR5fHvrIJaIHeAb0CBVlU1mnpHU0qUhAp3NcVHQtDM+ykcKvxjtNptRLp3o2AfTEPQmOHdG5KX21L69vagUWrwHf9mjxyQn6XfuzChj5W+fOcTlvHENK1Z8+KZg3a9sIhTFCPBxB4eX4TlUqS9HIndZFuiiDMszysJMZZ6wDDsUvlpdgHpSJlAmCoDd7cwjDWrTD//O3jRyYTbZTSFHpDRI/McnWJ7v8zYP0rlOQRqTb0=
```
Comment (fahhem) [inline .travis.yml]: I hope this is encrypted :)

Comment (jettisonjoe) [inline .travis.yml]: Yep, via travis's command-line tool.

File: .travis.yml (hunk 1 above)
Comment (jettisonjoe) [inline .travis.yml]: Travis's command-line tool did it! It's not my fault! I swear!

**Pull request merged**